shell/edit/provisioning.cattle.io.cluster/__tests__/DirectoryConfig.test.ts

```diff
@@ -0,0 +1,228 @@
+/* eslint-disable jest/no-hooks */
+import { mount, Wrapper } from '@vue/test-utils';
+import DirectoryConfig from '@shell/edit/provisioning.cattle.io.cluster/tabs/DirectoryConfig.vue';
+import { _EDIT, _CREATE } from '@shell/config/query-params';
+import { clone } from '@shell/utils/object';
+
+describe('component: DirectoryConfig', () => {
+  let wrapper: Wrapper<InstanceType<typeof DirectoryConfig>>;
+
+  const mountOptions = {
+    propsData: {
+      value: {
+        systemAgent:  '',
+        provisioning: '',
+        k8sDistro:    '',
+      },
+      mode: _CREATE,
+    },
+    mocks: {
+      $store: {
+        getters: {
+          'i18n/t':      jest.fn(),
+          'i18n/exists': jest.fn(),
+        }
+      },
+    }
+  };
+
+  it(`should render the component with its default configuration (create scenario)`, () => {
```

Comment:
no big deal since it's just in the unit tests, but you don't need js bakctick quotes.